mitosheet/mitosheet/tests/steps_performers/graph_steps/test_graph.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# Copyright (c) Saga Inc.
+# Distributed under the terms of the GPL License.
+
+import pandas as pd
+from mitosheet.step_performers.graph_steps.graph_utils import BAR, BOX, HISTOGRAM, SCATTER
+from mitosheet.tests.test_utils import create_mito_wrapper_dfs
+
+def test_create_graph_bar():
+    df = pd.DataFrame({'A': ['aaron', 'jake', 'nate'], 'B': [1, 2, 3]})
+    mito = create_mito_wrapper_dfs(df)
+    graph_id = '123'
+    mito.generate_graph(graph_id, BAR, 0, False, ['A'], ['B'], 400, 400)
+
+    assert len(mito.steps) == 2
+    assert mito.curr_step.step_type == 'graph'
+
+    graph_data = mito.get_graph_data(graph_id)
+
+    assert graph_data["graphParams"]["graphCreation"]["graph_type"] == 'bar'
+    assert graph_data["graphParams"]["graphCreation"]["sheet_index"] == 0
+    assert graph_data["graphParams"]["graphCreation"]["x_axis_column_ids"] == ['A']
+    assert graph_data["graphParams"]["graphCreation"]["y_axis_column_ids"] == ['B']
+
+    assert graph_data["graphOutput"]["graphScript"] is not None
+    assert graph_data["graphOutput"]["graphHTML"] is not None
+    assert graph_data["graphOutput"]["graphGeneratedCode"] is not None
+
+def test_create_graph_box():
+    df = pd.DataFrame({'A': ['aaron', 'jake', 'nate'], 'B': [1, 2, 3]})
+    mito = create_mito_wrapper_dfs(df)
+    graph_id = '123'
+    mito.generate_graph(graph_id, BOX, 0, False, ['A'], ['B'], 400, 400)
+    
+    assert len(mito.steps) == 2
+    assert mito.curr_step.step_type == 'graph'
+
+    graph_data = mito.get_graph_data(graph_id)
+
+    assert graph_data["graphParams"]["graphCreation"]["graph_type"] == 'box'
+    assert graph_data["graphParams"]["graphCreation"]["sheet_index"] == 0
+    assert graph_data["graphParams"]["graphCreation"]["x_axis_column_ids"] == ['A']
+    assert graph_data["graphParams"]["graphCreation"]["y_axis_column_ids"] == ['B']
+
+    assert graph_data["graphOutput"]["graphScript"] is not None
+    assert graph_data["graphOutput"]["graphHTML"] is not None
+    assert graph_data["graphOutput"]["graphGeneratedCode"] is not None
```

Comment:
Can you write a little helper function for getting at these values? Specifically: `getGraphType`, `getSheetIndex`, etc. These can live in the test utils file, but don't need to be defined on the class if you don't want - they can just be standalone.

This will allow us to:
1. Change the structure of this object
2. Only update where we change the structure in a single place to make the tests pass

Both are pretty important to me, because they make maintaining tests super easy to maintain! I've learned this lesson the hard way :-)